indico/modules/events/papers/templates/_contributions.html

```diff
@@ -31,6 +36,56 @@ <h3 class="icon-file">{% trans %}Paper{% endtrans %}</h3>
     {% endif %}
 {% endmacro %}
 
+{% macro render_editables_section(contribution) %}
+    {% set editable = contribution.editables[0] if contribution.editables if contribution.editables else None %}
```

Comment:
lowercase `none` in jinja

Reply:
you have `if contribution.editables` there twice

Reply:
eventually this needs to support multiple editables instead of doing the `editables[0]` 'hack' - maybe add a `{# TODO: ... #}` comment for this

Reply:
`{% set editable = contribution.editables|selectattr('type', 'equalto', 'paper')|first %}` probably works as well